.github/workflows/ci.yml

```diff
@@ -0,0 +1,20 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    name: Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # 2.3.4
```

Comment:
Hi @ErikSchierboom , just curious, why use the `@{sha}` instead of `@{semver}` ?
I see that dependabot updates to the latter...

Reply:
It's a security feature. If we pin to a tag (=version), we are potentially open to an attack where the tag is later removed upstream and re-added, but now linked to a malicious commit.